storage/filesystem/object.go

```diff
@@ -244,9 +244,9 @@ func (s *ObjectStorage) buildPackfileIters(
 	return iters, nil
 }
 
-type index map[plumbing.Hash]int64
+type idx map[plumbing.Hash]int64
```

Comment:
Collision with the `index` package, and this is the idx of the packfile and usually is called idx not index.